CONTRIBUTING.md

```diff
@@ -54,6 +48,15 @@ We are actively investigating ways of support enhancement requests in the projec
 
 Documentation is as important as code and we need your help to maintain clear and usable documentation.  If you find an error in here or other project documentation please [file an issue](https://github.com/devtools-html/debugger.html/issues/new) and tag it with the label [docs](https://github.com/devtools-html/debugger.html/labels/docs).
 
+### Share what you know
+
+You already know *far* more about the Debugger than the other people in your community.
+
```

Comment:
Not sure about the language here.  I see what you're getting at but I'd reword it.